packages/xsnap/scripts/get_xsnap_version.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
good enough, it assumes that `bash` is available, but I'll admit that's likely, and easier than figuring out how to write this in the purely `sh` subset

Reply:
Well arguably this is already posix 2022 compliant, it's just that most shells still don't support `-o pipefail` in their posix mode.

I suppose I could check for the existence of the binary first before trying to invoke it